sdk/monitor/opentelemetry-exporter-azuremonitor/tests/test_protocol.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import unittest
```

Comment:
Do we need this still?

Reply:
i think this can be removed

Reply:
We need it because we are using unittest asserts everywhere, if you are thinking we need to update all test cases to use pytest Asserts instead that is something we can address in different PR

Reply:
I was referring to `test_protocal.py` itself, since we are removing `protocol.py` and using `models.py` instead.